styles/variables-base.less

```diff
@@ -759,6 +759,14 @@
 @agate-switch-icon-position-left: 0;
 @agate-switch-selected-icon-position-left: (@agate-switch-width - @agate-switch-height);
 
+// SwitchItem
+// ---------------------------------------
+@agate-switchitem-icon-padding: 0;
+@agate-switchitem-icon-margin: 12px 0;
+@agate-switchitem-height: initial;
+@agate-switchitem-slotbefore-margin-right: 12px;
+@agate-switchitem-padding: 0 12px;
```

Comment:
I think it is better to use 0 @agate-item-side-outset and it will apply correctly also for gallium. Please delete this variable from variables-gallium.less